docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -710,7 +761,8 @@ Default stratifications to all observers should include scenario and input draw.
       * Probiotics availability
       * CPAP availability
       * Corticosteroid coverage
-    - Included except for corticosteroid coverage, which has not yet been added
+      * Believed gestational age
```

Comment:
This is a floating-point number, so I don't think we can stratify by this. Instead, I think we should make it a binary above/below some threshold.